src/main/resources/db/migration/V1_25_add_question_label_to_questions.sql

```diff
@@ -1,2 +1,2 @@
-ALTER TABLE question ADD COLUMN question_label VARCHAR(60);
-ALTER TABLE historical_question ADD COLUMN question_label VARCHAR(60);
+ALTER TABLE question ADD COLUMN question_label TEXT;
```

Comment:
The column name also has to match, e.g. being called `label` (not `question_label`). I suppose you can also simplify the name of the migration to `*_add_label_to_questions.sql`.

Could we also add the label to responses? Have we checked with Michael what's the intent of this change? Presumably it's to take a sort of snapshot of what was asked/answered (so including the response)